14-minimal/README.md

```diff
@@ -0,0 +1,250 @@
+NodeJS 14 container image
+=========================
+
+This container image includes Node.JS 14 as a [S2I](https://github.com/openshift/source-to-image) base image for your Node.JS 14 applications.
+Users can choose between RHEL, CentOS and Fedora based images.
+The RHEL images are available in the [Red Hat Container Catalog](https://access.redhat.com/containers/),
+the CentOS images are available on [Quay.io](https://quay.io/organization/centos7),
+and the Fedora images are available in [Fedora Registry](https://registry.fedoraproject.org/).
+The resulting image can be run using [podman](https://github.com/containers/libpod).
+
+Note: while the examples in this README are calling `podman`, you can replace any such calls by `docker` with the same arguments
+
+Description
```

Comment:
I would mentioned in description, what is difference between `nodejs-14` and `nodejs-14-minimal`

Reply:
Will add some simple info about the differences